doc/api/string_decoder.md

```diff
@@ -17,23 +23,49 @@ const euro = Buffer.from([0xE2, 0x82, 0xAC]);
 console.log(decoder.write(euro));
 ```
 
-## Class: StringDecoder
+When a `Buffer` instance is written to the `StringDecoder` instance, an
+internal buffer is used to ensure that the decoded string does not contain
+any partial multibyte-characters. These are held in the buffer until the
+next call to `stringDecoder.write()` or until `stringDecoder.end()` is called.
+
+For following example, for instance, writes the three bytes for the European
```

Comment:
`The following example` + I’d phrase it as `the three UTF-8 bytes` so that it’s clearer that the byte length of € does depend on the specific encoding